modules/internal/ChapelRange.chpl

```diff
@@ -830,17 +889,21 @@ module ChapelRange {
      if the represented sequence is not defined. */
   inline proc range.contains(ind: idxType)
   {
+    if chpl__singleValIdxType(idxType) {
+      if _low > _high then return false;
+    }
```

Comment:
Makes me want to have `proc range.isEmptySingleValueRange()`, as well as possibly (for other uses) `range.isSingleValueRange()`